common/src/main/kotlin/us/wedemy/eggeum/android/common/ui/BaseBottomSheetFragment.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Designed and developed by Wedemy 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/Wedemy/eggeum-android/blob/main/LICENSE
+ */
+
+package us.wedemy.eggeum.android.common.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.viewbinding.ViewBinding
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+abstract class BaseBottomSheetFragment<VB : ViewBinding> : BottomSheetDialogFragment() {
+  private var _binding: VB? = null
+  protected val binding: VB
+    get() = _binding!!
+
+  abstract fun getViewBinding(): VB
+
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?,
+  ): View = getViewBinding().also { _binding = it }.root
```

Comment:
```suggestion
  ) = getViewBinding().also { _binding = it }.root
```

저는 타입 명시를 지양하는 편이에요! smart-casting을 사용하지 않고 타입을 명시하면 우리가 관리해야 할 범위가 타입 명시부까지 확장되어서용 (반환 조건이 달라지면 타입 명시도 같이 수정해야 함)

Reply:
에고 여기에 타입을 냄겨놨네요 = 로 해놨는데 다시 바꾸겠슴니다~